### PR TITLE
quran v1.0.0 (new formulæ)

### PR DESCRIPTION
**Pull request by acagastya**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/quran.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "src/main.go"
+  end
+
+  test do
+    require "pty"
+    stdout, _stdin, _pid = PTY.spawn("#{bin}/quran 1:1")
```
Comment (carlocab) [inline Formula/quran.rb:17]: Does `shell_output` not work?

Comment (acagastya) [reply Formula/quran.rb:17]: I saw an old formulæ which used this. Should I rather use `shell_output`?

Comment (SMillerDev) [reply Formula/quran.rb]: That's the simplest/easiest way, yes

File: Formula/quran.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "src/main.go"
+  end
+
+  test do
+    require "pty"
+    op = shell_output("#{bin}/quran 1:1")
+    stripped_op = op.strip
```
Comment (SMillerDev) [inline Formula/quran.rb:18]:
```suggestion
    op = shell_output("#{bin}/quran 1:1").strip
```

**Review by carlocab — APPROVED** [review]
Thanks!

**Comment by chenrui333** [comment]
@acagastya, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**